HW1/P6/P6.txt

```diff
@@ -1 +1,25 @@
-# Your discussion here
+BY: MATHEUS C. FERNANDES - CS205 HW1 PROBLEM 6
+
+
+These are the 10 random Phrases of 20 words generated by the model:
+
+
+much more than the King. For what? He thinks, being twenty times so much as to say the truth, so 
+
+use. Here's Gloucester, a foe to th' purpose. Did he not in the world is not so much as to 
+
+boy. But come what will, I'll tarry no longer endure it; nor am sure you are a gentleman of the 
+
+that you have a good man and a good man and a good man and a good man and a 
+
+good now, ha! Would could meet that lucky ruler be employ'd, To show the world is not so much as 
+
+enough that you have a good man and a good man and a good man and a good man and 
+
+got so sweet a plant have you been all this while Between the King of Naples, To give me leave 
+
+murderers. Now show yourselves men; 'tis for a thousand pound a year; whose father died at Pomfret! Despair and die! 
+
+Exeunt Still, 'tis strange that desire should so dote on her back, And tell me what thou hast done a 
+
+Romeo.] Peter! Peter. Anon. Nurse. My fan, Peter. Mer. Good Peter, to hide my head. Hark, Tamora, the empress of 
```

Comment:
Great overall! In general, very succinct code, but you provide sufficient comments that describes your algorithm very well. However, in question 5, some of your functions are more complicated than the ones in other questions, and I feel you could have commented more.